app/models/event.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  has_many :user_events, foreign_key: :attended_event_id
+  has_many :attendees, through: :user_events, foreign_key: :attendee_id
+  default_scope -> { order(date_event: :asc) }
+  scope :upcoming, -> { where('date_event >= ?', Time.zone.now) }
```

Comment:
- [ ] Let's add validations to at least date_event because all your app depends on this attribute.

Reply:
Thanks for your useful recommendation. I added the validation.